.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -144,6 +144,12 @@ protected ParseResult ParseFraction(ExtractResult extResult)
                 demoPart = splitResult[1];
                 numPart = splitResult[2];
             }
+            else if (splitResult.Length == 1)
+            {
+                intPart = Config.ZeroChar.ToString(CultureInfo.InvariantCulture);
```

Comment:
What is this for? Can you add a code comment here with an example?

Reply:
comment added (needed to parse 'half' in Korean and Japanese).